hs_core/tasks.py

```diff
@@ -217,7 +218,7 @@ def send_over_quota_emails():
 
                 msg_str += '\n\nHydroShare Support'
                 subject = 'Quota warning'
-                if settings.DEBUG:
+                if settings.DEBUG or "beta" in Site.objects.get_current():
```

Comment:
@devincowan I am puzzled about why settings.DEBUG check itself would not be sufficient. ```settings.DEBUG``` should be set to True on beta, right?

Reply:
Indeed Beta should always be in debug mode, and the settings.DEBUG check should be adequate.
For some reason it wasn't last week and Beta sent out quota emails to 8 people.
So checking "beta" in the site URL is an additional measure in case the django debug mode gets turned off again...

Reply:
Just approved the PR. I guess for some reason settings.DEBUG was set to True accidentally last week for some reason. Having an additional measure is good, but note other test VMs than beta are not included in the additional check. In any case, such accidents should be very rare and hopefully will never happen again on any test servers.

Reply:
Good point Hong, thanks for that catch! I just updated so that quota emails are prevented from any url not containing www.hydroshare.org